script/RumpelConfig.sol

```diff
@@ -4043,6 +4052,27 @@ library RumpelConfig {
             approveAllowState: RumpelGuard.AllowListState.OFF
         });
 
+        return configs;
+    }
+
+        function getHyperevmPendleClaimUpdateProtocolConfigs()internal
+        pure
+        returns (ProtocolGuardConfig[] memory)
+    {
+        ProtocolGuardConfig[] memory configs = new ProtocolGuardConfig[](2);
+
+        configs[0] = ProtocolGuardConfig({target: MAINNET_PENDLE_ROUTERV4, selectorStates: new SelectorState[](1)});
+        configs[0].selectorStates[0] = SelectorState({
+            selector: IPendleRouterV4.redeemDueInterestAndRewardsV2.selector,
+            state: RumpelGuard.AllowListState.OFF
+        });
+
+        configs[1] = ProtocolGuardConfig({target: MAINNET_PENDLE_ROUTERV4, selectorStates: new SelectorState[](1)});
+        configs[1].selectorStates[0] = SelectorState({
+            selector: ActionMiscV3_RumpelV2.redeemDueInterestAndRewardsV2.selector,
```

Comment:
So the main difference between this and IPendleRouterV4's version is the SwapDataExtra, right?

And Mainnet has this one (ActionMiscV3_RumpelV2), but not the IPendleRouterV4 version? (from a quick look at mainnet state)